homework_7/hw_fun.py

```diff
@@ -0,0 +1,80 @@
+import sys
+
+
+# Sequential map
+
+def sequential_map(*args) -> list:
+    args = [*args]
+    cont = args.pop()
+    for fun in args:
+        cont = list(map(fun, cont))
+    return cont
```

Comment:
Превратить элемент в список можно только в самом конце, map можно передавать в другие map'ы
```suggestion
    for fun in args:
        cont = map(fun, cont)
    return list(cont)
```